monorepo/website/src/pages/docs/how-to/revise-submissions.mdx

```diff
@@ -17,6 +17,8 @@ Submitting these changes ("revisions") will cause the version of the sequence to
 The process of submitting revisions is very similar to original submission.
 The main difference is that you must provide an `accession` column in the metadata file, which contains the Pathoplexus accession number(s) assigned when the sequences were submitted originally.
 
+**You must use the _original metadata_ when doing a revision!** (In other words, the original files you submitted to Pathoplexus.) Currently you will need to have this file locally - we aim to introduce a way to download this from Pathoplexus soon. If you do not have these files, please _contact us_ to help you do the revision.
+

```

Comment:
or wait should we give an email to contact us at?

Reply:
Ah good point. I can add this!

Reply:
I also fixed the going back and forth between plural and singular 'file(s)'. What do you think?